docker/iml-mailbox.dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM rust-iml-base as builder
+
+FROM imlteam/systemd-base
+COPY --from=builder /build/target/release/iml-mailbox /bin/
+COPY docker/iml-mailbox/iml-mailbox.service /etc/systemd/system/
+COPY docker/iml-mailbox/iml-mailbox.conf /etc/systemd/system/
+COPY iml-mailbox.conf /usr/lib/tmpfiles.d/
+COPY docker/wait-for-dependencies.sh /usr/local/bin/
+
+RUN systemctl enable iml-mailbox
```

Comment:
It does not require systemd daemon running etc., does it?

Reply:
It's available to enable the service at this point. It's actually really neat that we can do this inside a container. Here's the link for more info in case you're interested: https://developers.redhat.com/blog/2016/09/13/running-systemd-in-a-non-privileged-container/